src/shared_modules/utils/stringHelper.h

```diff
@@ -25,6 +25,74 @@
 
 namespace Utils
 {
+    static bool patternMatch(const std::string& entryName, const std::string& pattern)
+    {
+        auto match { true };
+        // Match the glob pattern without regex
+        auto patternPos { 0u };
+
+        for (auto i { 0u }; i < entryName.size(); ++i)
+        {
+            if (patternPos < pattern.size())
+            {
+                // 'x' matches 'x'
+                if (entryName.at(i) == pattern.at(patternPos))
```

Comment:
Wildcard characters "*" and "?" are allowed in the entryName string?
If yes, an escape character should be evaluated to avoid issues in this conditional.